### PR TITLE
show invited users in invite responsible field

### PR DESCRIPTION
**Pull request by rahul2240**
This PR adds the following features -
1. Invited users who have not joined the app will be shown in invite responsible field.
2. The submitter will be able to remove the invited user from the edit page.

**Showing invited users in invite responsible field**
showinvited

### CODE REVIEW

**Comment by differentreality** [comment]
Hmmm I just realized it is easy to think you have added an email to invite, when you actually haven't. Wonder if we could take care of that. 

On a different note, booths#show could also show the invited users?

**Comment by rahul2240** [comment]
I will think about that and yes we can show invited users in booths#show.

**Comment by differentreality** [comment]
What does the admin see about such a booth?

**Comment by rahul2240** [comment]
Sorry I didn't get you.

**Comment by differentreality** [comment]
I meant, please add a screenshot for the organizers point of view.
Where do the invited users show up for the organizer (in admin side of the application)?

**Comment by rahul2240** [comment]
Sorry for replying late. Right now! we are showing responsibles in booths#show and booths#edit. For the admins, I will create a new page that will render all invited responsibles.

File: app/controllers/admin/booths_controller.rb [hunk 1]
```diff
@@ -50,6 +50,7 @@ def create
 
     def edit
       @url = admin_conference_booth_path(@conference.short_title, @booth.id)
+      @invitation_pending_emails = @booth.responsibles.where(last_sign_in_at: nil).pluck(:email).join(',')
```
Comment (differentreality) [inline app/controllers/admin/booths_controller.rb]: Should we extract this to a booth method `invited_responsibles` or something like that?

**Comment by differentreality** [comment]
Yeah it would be useful for organizer to know what's going on as well. 
Other than that, not sure why this is still a WIP?

File: app/controllers/admin/booths_controller.rb (hunk 1 above)
Comment (rahul2240) [reply app/controllers/admin/booths_controller.rb]: Yes, we can do that.

**Comment by rahul2240** [comment]
@differentreality I have made the changes. Please review.

**Pull request closed without merging**